addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ApplicationErrorScannerUnitTest.java

```diff
@@ -41,6 +45,24 @@ protected ApplicationErrorScanner createScanner() {
         return new ApplicationErrorScanner();
     }
 
+    @Override
+    public void setUpZap() throws Exception {
+        super.setUpZap();
+
+        Path xmlDir = Files.createDirectories(Paths.get(Constant.getZapHome(), "xml"));
+        Path testFile = xmlDir.resolve("application_errors.xml");
+        String content =
+                "<?xml version=\"1.0\" standalone=\"no\"?>\n"
+                        + "<!-- \n"
+                        + "UnitTest File\n"
+                        + "-->\n"
+                        + "<Patterns>\n"
+                        + "  <Pattern type=\"string\">Microsoft OLE DB Provider for ODBC Drivers</Pattern>\n"
+                        + "  <Pattern type=\"regex\">(?i)Line\\s\\d+:\\sIncorrect\\ssyntax\\snear\\s'[^']*'</Pattern>\n"
+                        + "</Patterns>";
+        Files.write(testFile, content.getBytes());
```

Comment:
Better to get the bytes with `StandardCharsets.UTF_8` (that's what will be used to read the file).